upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -442,7 +442,7 @@ state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
 if [[ ${state_recorded} == "0" && $(hostname) == "ncn-m001" ]]; then
     echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
     {
-    
+        helm -n argo upgrade --install cray-iuf "${CSM_ARTI_DIR}/helm/cray-iuf-0.0.1.tgz"
```

Comment:
What happens when the IUF version changes? Should we create a preupgrade loftsman manifest in the CSM repo instead?

Reply:
https://github.com/Cray-HPE/csm/tree/main/manifests - upgrade-me-first.yml

Reply:
in talking with di -- nls is used in prereqs, and that depends on iuf.  so moving out of prereq's isn't an option..

Reply:
But would it make sense to run a Loftsman command here?

Reply:
Merging this. Looking for a small refactor so that versions of artifacts are colocated out of the CSM repo going forward.